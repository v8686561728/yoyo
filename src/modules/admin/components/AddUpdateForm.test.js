// import React from "react";
// import { shallow } from "../../../enzyme";
// import { shallowToJson } from "enzyme-to-json";
// import { AddUpdateForm,mapStateToProps } from "./AddUpdateForm";

// describe("rendering", () => {
//   let wrapper;
//   let props = {
//     match: {
//       params: "1"
//     },
//     giftCards: [
//       {
//         id: "2"
//       }
//     ]
//   };
//   let props2 = {
//     match: {
//       params: "1"
//     },
//     giftCards: [
//       {
//         id: "1"
//       }
//     ]
//   };
//   it("Should render AddUpdateForm component", () => {
//     wrapper = shallow(<AddUpdateForm {...props} />);
//     expect(shallowToJson(wrapper)).toMatchSnapshot();
//   });

//   it("Should have unequal id", () => {
//     wrapper = shallow(<AddUpdateForm {...props2} />);
//     expect(shallowToJson(wrapper)).toMatchSnapshot();
//   });

//   it("should map state to props", () => {
//     const initialState = {
//       gifts: {
//         giftCards: []
//       }
//     };
//     expect(mapStateToProps(initialState).giftCards).toEqual([]);
//   });
// });