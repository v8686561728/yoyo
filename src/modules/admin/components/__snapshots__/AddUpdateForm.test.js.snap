// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering Should have unequal id 1`] = `
<Fragment>
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Name"
    inputType="text"
    inputValue=""
    name="NAME"
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Points"
    inputType="text"
    inputValue=""
    name="POINTS"
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Category"
    inputType="text"
    inputValue=""
    name="CATEGORY"
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Retailer"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Expiry Date"
    inputType="date"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Count"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Image"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Vendor"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Short Description"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Long Description"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <div
    style={
      Object {
        "marginBottom": "100px",
        "marginTop": "10px",
      }
    }
  >
    <WithStyles(ForwardRef(Button))
      onClick={[Function]}
      style={
        Object {
          "marginLeft": "20px",
        }
      }
      variant="contained"
    >
      CLEAR
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="primary"
      onClick={[Function]}
      style={
        Object {
          "marginLeft": "10px",
        }
      }
      variant="contained"
    >
      ADD
    </WithStyles(ForwardRef(Button))>
  </div>
  <div />
</Fragment>
`;

exports[`rendering Should render AddUpdateForm component 1`] = `
<Fragment>
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Name"
    inputType="text"
    inputValue=""
    name="NAME"
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Points"
    inputType="text"
    inputValue=""
    name="POINTS"
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Category"
    inputType="text"
    inputValue=""
    name="CATEGORY"
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Retailer"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Expiry Date"
    inputType="date"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Count"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Image"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Card Vendor"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Short Description"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <InputTypeComponent
    endAdornment={false}
    handleInputChange={[Function]}
    inputError={false}
    inputHelperText=""
    inputPlaceholder="Long Description"
    inputType="text"
    inputValue=""
    styles={
      Object {
        "marginBottom": "1rem",
        "marginLeft": "1rem",
        "marginTop": "1rem",
        "width": "45%",
      }
    }
  />
  <div
    style={
      Object {
        "marginBottom": "100px",
        "marginTop": "10px",
      }
    }
  >
    <WithStyles(ForwardRef(Button))
      onClick={[Function]}
      style={
        Object {
          "marginLeft": "20px",
        }
      }
      variant="contained"
    >
      CLEAR
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="primary"
      onClick={[Function]}
      style={
        Object {
          "marginLeft": "10px",
        }
      }
      variant="contained"
    >
      ADD
    </WithStyles(ForwardRef(Button))>
  </div>
  <div />
</Fragment>
`;
